package.json

```diff
@@ -40,16 +40,16 @@
     "url": "https://github.com/synacor/eslint-config-synacor.git"
   },
   "dependencies": {
-    "babel-eslint": "^10.0.1",
-    "eslint-plugin-mocha": "^5.3.0",
-    "eslint-plugin-react": "^7.12.4"
+    "babel-eslint": "^10.0.2",
+    "eslint-plugin-mocha": "^6.0.0",
+    "eslint-plugin-react": "^7.14.3"
   },
   "devDependencies": {
-    "eslint": "^5.16.0",
-    "rollup": "^1.8.0",
-    "rollup-plugin-buble": "^0.19.6"
+    "eslint": "^6.1.0",
+    "rollup": "^1.18.0",
+    "rollup-plugin-buble": "^0.19.8"
   },
   "peerDependencies": {
-    "eslint": "^4.0.0 || ^5.0.0"
+    "eslint": "^4.0.0 || ^5.0.0 || ^6.0.0"
```

Comment:
If we are going to include eslint 6.0, we'll need to modify the rules in this package to disable some new default rules that could be a breaking change for people

Reply:
sure I'll rework on it then